modules/apps/marketplace/marketplace-app-manager-web/src/main/java/com/liferay/marketplace/app/manager/web/internal/display/context/BaseAppManagerManagementToolbarDisplayContext.java

```diff
@@ -67,12 +72,22 @@ public String getCategory() {
 		String[] categories = MarketplaceAppManagerUtil.getCategories(
 			apps, bundles);
 
+		String[] translatedCategories = new String[categories.length];
+
+		for (int i = 0; i < categories.length; i++) {
+			String kebabCaseCategory = StringUtil.replace(
+				categories[i].toLowerCase(), CharPool.SPACE, CharPool.DASH);
+
+			translatedCategories[i] = LanguageUtil.get(
+				request, kebabCaseCategory, categories[i]);
+		}
+
 		PortletURL portletURL = getPortletURL();
 
 		portletURL.setParameter("resetCur", Boolean.TRUE.toString());
 
 		return getDropdownItems(
-			getDefaultEntriesMap(categories), portletURL, "category",
+			getDefaultEntriesMap(translatedCategories), portletURL, "category",
```

Comment:
We have to construct our own map here, since the values should still be in English (whatever was found in the `bnd.bnd` file).  You could just do it inside the loop (instead of creating a second array).